src/analysis.py

```diff
@@ -13,6 +13,9 @@
 from pyulog.core import ULog
 from pyqtgraph.Qt import QtCore,QtGui
 import pyqtgraph as pg
+from matplotlib.backends.backend_qt5agg import FigureCanvasQTAgg as FigureCanvas
```

Comment:
You add this, but it seems that you didn't use it.

Reply:
I'm sorry it is my mistake.